tb_plugin/torch_tb_profiler/plugin.py

```diff
@@ -346,7 +346,11 @@ def _get_run_dirs(self):
             /run2
                 /[worker1].pt.trace.json
         """
-        for root, _, files in io.walk(self.logdir):
+        # Note on followlinks=True: per the tensorboard documentation [1], users are encouraged to
+        # use symlink trees to have fine-grained control over the filesystem layout of runs. To
+        # support such trees, we must follow links.
+        # [1] https://github.com/tensorflow/tensorboard/blob/master/README.md#logdir--logdir_spec-legacy-mode
+        for root, _, files in io.walk(self.logdir, followlinks=True):
```

Comment:
It seems only need change the os.walk call since the cloud storage doesn't have the symlink concept?

Reply:
Indeed that would make this change much less invasive! I wasn't sure if it was acceptable to enable `followlinks` all the time without giving a choice to the caller; but it appears that currently there's only one caller anyway.

I'll make the change!